microsoft-edge/webview2/samples/webview2browser.md

```diff
@@ -87,6 +87,8 @@ int APIENTRY wWinMain(_In_ HINSTANCE hInstance,
     // ...
 ```
 
+<!-- todo: in this PR, remove "win 7" comment above?  Remove the below step & code listing?? -->
```

Comment:
@nishitha-burman - click expanders in Files Changed PR page to see code listing above & below.

Reply:
Retained per Dev; no change to this .md